picard/tagger.py

```diff
@@ -412,7 +412,32 @@ def process(result=None, error=None):
                     self.add_files(result)
                 thread.run_task(get_files, process)
 
-        process(True, False)
+        def get_files_non_recursive():
+            listings = [f for f in os.listdir(path)]
```

Comment:
I think it would be better to not use all these lambdas here, but instead just use a single loop:

```python
files = []
for filename in os.listdir(path):
   join()
   if is_hidden() or not isfile():
     continue
   files.append()
```